lib/cleanup/destroyable_records.rb

```diff
@@ -58,15 +58,17 @@ def print_data
   end
 
   def destroy_records
-    stdout.puts 'Destroying integration usages'
-    integration_usages.each do |integration_usage|
-      integration_usage.destroy!
-    end
-    integration.reload
+    if integration
+      stdout.puts 'Destroying integration usages'
+      integration_usages.each do |integration_usage|
+        integration_usage.destroy!
+      end
```

Comment:
what if we did this in one pass?
```suggestion
      integration_usages.destroy_all
```

Reply:
its possible for `integration_usages` to be an empty array

Reply:
looks like this PR undid that by removing the `|| []`

Reply:
okay yes you're right, I added another test for integrations with no integration usages because it looked as though it was returning an empty array in that case even without that bit of code but the test still passed with `destroy_all` so I think we're good

Reply:
I removed the `|| []` because Integration `has_many`  integration_usages and rails makes that an empty array automatically - see [source code](https://github.com/18F/identity-idp/blob/c099dc78a30532e7d473156976fabbc15c8907ac/app/models/agreements/integration.rb#L8)